vercel.json

```diff
@@ -7,6 +7,10 @@
   },
   "cleanUrls": true,
   "rewrites": [
+    {
+      "source": "/v/:version/:asset*",
+      "destination": "https://www.unpkg.com/@seamapi/react@:version/:asset"
+    },
```

Comment:
can't remember if this works

```suggestion
    {
      "source": "/v/:version/:asset*",
      "destination": "https://www.unpkg.com/@seamapi/react@:version/:asset",
      "headers": [
          {
            "key": "Cache-Control",
            "value": "s-maxage= 604800"
          }
        ]
    },
```

Reply:
i think it should https://vercel.com/guides/how-to-configure-the-cache-control-response-header-in-vercel-projects#using-the-vercel.json-file

Reply:
hmm, unclear, but worth noting that it'll make things faster if we can use the Cache-Control header

Reply:
We should! Good call! There are a standard set of "Asset CDN" headers I like to ensure are set and forgot to consider them given we are proxing. I think in this case we can respect unpkg + Vercel for now as they seem to be adding this for us without additional config:

<img width="382" alt="image" src="https://github.com/seamapi/react/assets/721372/2b3e7fb4-57cc-466a-8dcf-3d34b56c1637">

<img width="817" alt="image" src="https://github.com/seamapi/react/assets/721372/5e773739-5905-4b5d-b021-dfaa1d03472d">

Reply:
Actually, I am more concerned about everything else (that is not an immutable asset), so made an issue to review headers later: https://github.com/seamapi/react/issues/186

Reply:
hmm, Vercel couldn't be adding Cache-Control automatically

Reply:
oh i think i see nvm

Reply:
So unpkg adds cache-control, Vercel sees that and it's the same as us providing it. Gotcha